test/utils/UintUtils.ts

```diff
@@ -31,11 +31,74 @@ describe('UintUtils', function () {
 
     describe('#toHexString', function () {
       describe('(uint256)', function () {
-        it('todo');
+        it('Should return 0 if input is 0', async () => {
```

Comment:
```suggestion
        it('returns 0 if input is 0', async () => {
```
The word "should" is often used in test descriptions, but it only carries meaning in the context of test/spec religious dogma.  Better to use the action verb directly.